src/Metadata/PropertyTypeIterable.php

```diff
@@ -12,8 +12,18 @@
  *  - we're not merging a plain array PropertyTypeIterable into a hashmap one,
  *  - and the collection classes of each are either not present on both sides, or are the same, or parent-child of one another
  */
-final class PropertyTypeIterable extends PropertyTypeArray
+final class PropertyTypeIterable extends AbstractPropertyType
 {
+    /**
+     * @var PropertyType
+     */
+    protected $subType;
```

Comment:
```suggestion
    private PropertyType $subType;
```

Reply:
i think it makes sense to change it for 2.x, but we should do a rectorphp run for that rather than changing it one-by-one. so for consistency its fine to not have the type declaration in this pull request.